### PR TITLE
More details in the coordinated shutdown docs

### PR DESCRIPTION
**Pull request by johanandren**

### CODE REVIEW

**Comment by johanandren** [comment]
PLS BUILD

**Review by patriknw — APPROVED** [review]
LGTM

File: akka-actor/src/main/resources/reference.conf [hunk 1]
```diff
@@ -1080,17 +1087,22 @@ akka {
       before-service-unbind {
       }
 
-      # Stop accepting new incoming requests in for example HTTP.
+      # Stop accepting new incoming requests.
```
Comment (ktoso) [inline akka-actor/src/main/resources/reference.conf]: "Stop accepting new incoming connections" would be more real wording here.
Our unbind does not kill existing connections (currently);

It should be implemented in https://github.com/akka/akka-http/issues/1210 as we don't do it nowadays still.

See also  https://github.com/akka/akka-http/issues/575

File: akka-actor/src/main/resources/reference.conf [hunk 2]
```diff
@@ -1080,17 +1087,22 @@ akka {
...
+      # This is where you can register tasks that makes a server stop accepting new connections. Already
```
Comment (ktoso) [inline akka-actor/src/main/resources/reference.conf]: yeah here the wording is good

**Review by ktoso — APPROVED** [review]
LGTM with one rewording

**Pull request merged**